<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="volumeBar">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="mute">
       <property name="styleSheet">
        <string notr="true">background: transparent;
        border: none;
        padding: 0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/volume_.png</normaloff>:/img/volume_.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>411</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="c4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#c4{
 border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#c4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Q</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#d4{
 border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#d4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>W</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="e4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#e4{
        border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#e4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>E</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="f4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#f4{
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                  stop:0 #8e2de2, stop:1 #4a00e0);
        border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#f4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>R</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="g4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#g4{
 border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#g4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>T</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="a4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#a4{
 border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#a4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Y</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#b4{
 border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#b4:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>U</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#c5{
 border: 2px solid #e2e2e2;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(226, 226, 226, 255));
}
#c5:pressed{
background-color: rgb(180, 180, 180);
        border: 2px solid #e2e2e2
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="shortcut">
        <string>I</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>111</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="c_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#c_4{
        border: 2px solid #121212;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(12, 12, 12, 255), stop:1 rgba(100, 100, 100, 255));
}
#c_4:pressed{
background-color: rgb(0, 0, 0);
        border: 2px solid #000000
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/bkey.png</normaloff>:/bkey.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>99</height>
        </size>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#d_4{
        border: 2px solid #121212;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(12, 12, 12, 255), stop:1 rgba(100, 100, 100, 255));
}
#d_4:pressed{
background-color: rgb(0, 0, 0);
        border: 2px solid #000000
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/bkey.png</normaloff>:/bkey.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>99</height>
        </size>
       </property>
       <property name="shortcut">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>160</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="f_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#f_4{
        border: 2px solid #121212;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(12, 12, 12, 255), stop:1 rgba(100, 100, 100, 255));
}
#f_4:pressed{
background-color: rgb(0, 0, 0);
        border: 2px solid #000000
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/bkey.png</normaloff>:/bkey.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>99</height>
        </size>
       </property>
       <property name="shortcut">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="g_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#g_4{
        border: 2px solid #121212;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(12, 12, 12, 255), stop:1 rgba(100, 100, 100, 255));
}
#g_4:pressed{
background-color: rgb(0, 0, 0);
        border: 2px solid #000000
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/bkey.png</normaloff>:/bkey.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>99</height>
        </size>
       </property>
       <property name="shortcut">
        <string>6</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="a_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#a_4{
        border: 2px solid #121212;
        border-radius: 8px;
        padding: 8px;
        color: white;
        font-size: 14px;
background: qlineargradient(spread:pad, x1:0.51, y1:0, x2:0.533, y2:1, stop:0 rgba(12, 12, 12, 255), stop:1 rgba(100, 100, 100, 255));
}
#a_4:pressed{
background-color: rgb(0, 0, 0);
        border: 2px solid #000000
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/bkey.png</normaloff>:/bkey.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>99</height>
        </size>
       </property>
       <property name="shortcut">
        <string>7</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
     <item>
      <widget class="QComboBox" name="instrumentBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="octaveBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="gotoButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
        border: none;
        padding: 0px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/goto_.png</normaloff>:/img/goto_.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
        border: none;
        padding: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/info_.png</normaloff>:/img/info_.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
